### PR TITLE
Doc/13 upload response

### LINKED ISSUE #13: Document what's the resul of upload
Document what returns image server for upload request and if it could be somehow used in the app.

### PR DESCRIPTION
**Pull request by horaklukas**
Document what's the response of upload object.

Resolves #13

### CODE REVIEW

File: packages/node-resizin/README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ upload(file).then(() => {
 *  There is also third parametr, available only for Node.js, that determines file mime type. The mime type is optional and defaults to `application/octet-stream`.
```
Comment (baierjak) [inline packages/node-resizin/README.md:51]: old error but => parameter

File: packages/resizin/package.json [hunk 2]
```diff
@@ -34,6 +34,6 @@
     "isomorphic-fetch": "3.0.0",
     "lodash": "^4.17.20",
     "promise-polyfill": "^8.1.3",
-    "uuid": "^3.3.3"
+    "uuid": "^8.3.2"
```
Comment (baierjak) [inline packages/resizin/package.json:37]: Recently I have read some articles that this UUID library is not so good for performance..... as an alternative is highly recommended to use NANOID - https://openbase.com/js/nanoid but I have never tried it before :)

Comment (baierjak) [reply packages/resizin/package.json:37]: maybe in that newest version the uuid is fixed

File: packages/node-resizin/README.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ yarn add node-resizin
 ## Quick start
 
 ```js
-const { uploadFactory } = require('node-resizin');
+const uploadFactory = require('node-resizin');
```
Comment (baierjak) [inline packages/node-resizin/README.md:29]: this import is changed from named import to default import - but I cant see any change in the code or update version in package.json

It was an old mistake from before just here in the documentation or what?

File: packages/node-resizin/README.md [hunk 4]
```diff
@@ -51,7 +51,7 @@ upload(file).then(() => {
 *  There is also third parametr, available only for Node.js, that determines file mime type. The mime type is optional and defaults to `application/octet-stream`.
 
     ```js
-    const { uploadFactory } = require('node-resizin');
+    const uploadFactory = require('node-resizin');
```
Comment (baierjak) [inline packages/node-resizin/README.md:54]: this import is changed from named import to default import - but I cant see any change in the code or update version in package.json

It was an old mistake from before just here in the documentation or what?

**Review by baierjak — APPROVED** [review]

**Comment by baierjak** [comment]
Good job, pretty nice to have it well documented :)

File: packages/node-resizin/README.md (hunk 3 above)
Comment (horaklukas) [reply packages/node-resizin/README.md:29]: It was a bug in documentation caused probably by Copy/Paste. If you're importing uploadFactory from `resizin` package it's named import, but from `node-resizin` it's default import as there is nothing else in node package.

File: packages/resizin/package.json (hunk 2 above)
Comment (horaklukas) [reply packages/resizin/package.json:37]: Good point, will check it out 👍

File: packages/node-resizin/README.md (hunk 1 above)
Comment (horaklukas) [reply packages/node-resizin/README.md]: fixed

**Pull request merged**